### DISCUSSION TITLE
Callback function help; calling forever

### DISCUSSION THREAD

**Discussion by GioByte10** [versa-syahptr/winotify#25] [Q&A]
I'm trying to use the callback feature of winotify. The callback function is executed when the user presses the button, but the notification keeps on reappearing if I keep clicking the button. How can I make sure that after I click the button, the notification doesn't reappear? Perhaps I am using the callback function wrong. I attach the code I'm testing.

```
import winotify

r = winotify.Registry("test2", winotify.PY_EXE, __file__)
notifier = winotify.Notifier(r)


@notifier.register_callback
def callback():
    print("callback")


if __name__ == '__main__':
    toast = notifier.create_notification(title="Notification",
                                         msg='Notification test with callback')

    toast.add_actions(label="say callback in console",
                      launch=callback)

    notifier.start()
    toast.show()


```

**Comment by versa-syahptr** [comment]
You have to call `notifier.start` before everything else
```python
if __name__ == '__main__':
    notifier.start()
```
